assets/js/modules/analytics-4/datastore/conversion-events.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * `modules/analytics-4` data store: coversion events.
+ *
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import invariant from 'invariant';
+
+/**
+ * Internal dependencies
+ */
+import API from 'googlesitekit-api';
+import Data from 'googlesitekit-data';
+import { MODULES_ANALYTICS_4 } from './constants';
+import { createFetchStore } from '../../../googlesitekit/data/create-fetch-store';
+import { createReducer } from '../../../googlesitekit/data/create-reducer';
+
+const fetchGetConversionEventsStore = createFetchStore( {
+	baseName: 'getConversionEvents',
+	controlCallback: ( { propertyID } ) => {
+		return API.get(
+			'modules',
+			'analytics-4',
+			'conversion-events',
+			{
+				propertyID,
+			},
+			{
+				useCache: false,
+			}
```

Comment:
I think this should be omitted to use the default caching behaviour like report and goal requests do.

Reply:
Updated 👍